examples/workspace/Untitled 1.txt

```diff
@@ -0,0 +1 @@
+sasdsdfasdadasda asd 
```

Comment:
I think this is committed on accident?

Reply:
Same is true for "Untitled.txt" and "asda" and "test"

Reply:
:sweat: Yes, no idea how these files ended up there :grin: